coremltools/converters/mil/mil/passes/use_reflection_padding.py

```diff
@@ -73,6 +72,9 @@ def _match_pattern(concat_op, block):
 
         slice_ops_out.append(slice_op)
 
+    if axis is None:
+        return False
```

Comment:
I'm not 100% sure this will fix the bug in issue #1376 

It will certainly fix the python interpreter error on line 78 ( local variable 'axis' referenced before assignment) but I think it will return False instead of the op

Reply:
Returning `False` here means the pass/optimization should not be applied.

It does fix #1376. I've verified that.